src/main/java/io/quarkus/gizmo2/creator/ops/ListOps.java

```diff
@@ -29,4 +29,14 @@ public ListOps(final BlockCreator bc, final Expr obj) {
     protected ListOps(final Class<?> receiverType, final BlockCreator bc, final Expr obj) {
         super(receiverType.asSubclass(List.class), bc, obj);
     }
+    
+    /**
+     * Generate a call to {@link List#get(int)}.
+     *
+     * @param index the index of the element to return (must not be {@code null})
+     * @return the expression of the result (not {@code null})
+     */
+    public Expr get(Expr index) {
```

Comment:
I would add an overload which accepts an `int`-typed index for this.

Reply:
Done.